### ISSUE TITLE
Colors match when they should not

### ISSUE THREAD

**Issue by jesshart** [pcottle/learnGitBranching#986]
First of all, incredible work here. A fantastic web app for learning Git.

It looks like colors may be randomly rendered for examples to distinguish one branch from another. However, sometimes the same color is rendered for different branches:
![Screen Shot 2022-07-22 at 1 43 38 PM](https://user-images.githubusercontent.com/59590184/180504690-dffed6b4-125c-4af6-b2e8-e4c62c233813.png)

**Comment by pcottle** [comment]
Yeah there's a chance, I think I didn't handle the collision case. Open to reviewing a PR if you have any ideas though!

**Comment by AjayLiu** [comment]
I was trying to see if I could tinker with the color generation but it's crazy! Note to those who want to try this, the color generation is in `src/visuals/index.js`, not `src/visuals/visBranch.js` like I thought for a good 2 hours lol!

I saw comment in the code and it gave me a good chuckle:
```js
// crap! convert
var color = Raphael.color(fill);
fill = 'hsb(' + String(color.h) + ',' + String(color.l);
fill = fill + ',' + String(color.s) + ')';
```
I'm new to open source but I'd love to contribute to this project! I'm just a little confused since the colors get converted and blended so much lol
